lib/utils/index.ts

```diff
@@ -18,21 +18,99 @@ import {
 import type { Rule, AST, SourceCode } from "eslint"
 import { parseStringTokens } from "./string-literal-parser"
 import { findVariable } from "./ast-utils"
-import type { ReadonlyFlags } from "regexp-ast-analysis"
+import type { ReadonlyFlags, ToCharSetElement } from "regexp-ast-analysis"
+// eslint-disable-next-line no-restricted-imports -- Implement RegExpContext#toCharSet
+import { toCharSet } from "regexp-ast-analysis"
+import type { CharSet } from "refa"
 export * from "./unicode"
 
+export type ToCharSet = (
+    node: ToCharSetElement,
+    flags?: ReadonlyFlags,
+) => CharSet
+type RegExpHelpersBase = {
+    toCharSet: ToCharSet
+
+    /**
+     * Creates SourceLocation from the given regexp node
+     * @see getRegexpLocation
+     *
+     * @param regexpNode The regexp node to report.
+     * @param offsets The report location offsets.
+     * @returns The SourceLocation
+     */
+    getRegexpLocation: (
+        regexpNode: RegExpNode,
```

Comment:
Right now, we have `import type { Node, Node as RegExpNode } from "regexpp/ast"`. Could you please remove either `Node` or `Node as RegExpNode`?

Reply:
Thank you for letting me know! I fixed it.